### PR TITLE
modules/py_image: Add unpacking to bytearray support.

### PR DESCRIPTION
**Pull request by kwagyeman**

### CODE REVIEW

File: src/omv/modules/py_image.c [hunk 1]
```diff
@@ -724,6 +724,79 @@ static mp_obj_t py_image_bytearray(mp_obj_t img_obj) {
 }
 static MP_DEFINE_CONST_FUN_OBJ_1(py_image_bytearray_obj, py_image_bytearray);
 
+static mp_obj_t py_image_unpack(uint n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
+    enum { ARG_buffer, ARG_dtype, ARG_scale, ARG_mean, ARG_stdev };
+    static const mp_arg_t allowed_args[] = {
+        { MP_QSTR_buffer, MP_ARG_OBJ | MP_ARG_REQUIRED, {.u_rom_obj = MP_ROM_NONE} },
+        { MP_QSTR_dtype, MP_ARG_OBJ | MP_ARG_REQUIRED, {.u_rom_obj = MP_ROM_NONE } },
```
Comment (iabdalkader) [inline src/omv/modules/py_image.c:731]:
```suggestion
        { MP_QSTR_dtype, MP_ARG_INT | MP_ARG_REQUIRED, {.u_int = 'B' } },
```

File: src/omv/modules/py_image.c [hunk 2]
```diff
@@ -724,6 +724,79 @@ static mp_obj_t py_image_bytearray(mp_obj_t img_obj) {
...
+    mp_arg_parse_all(n_args - 1, pos_args + 1, kw_args, MP_ARRAY_SIZE(allowed_args), allowed_args, args);
+
+    mp_buffer_info_t bufinfo = {0};
+    mp_get_buffer_raise(args[ARG_buffer].u_obj, &bufinfo, MP_BUFFER_WRITE);
+
+    // The first character is either 0 or the typecode...
+    const char *dtype = mp_obj_str_get_str(args[ARG_dtype].u_obj);
+
+    int dtype_size;
+    switch (dtype[0]) {
```
Comment (iabdalkader) [inline src/omv/modules/py_image.c:748]:
```suggestion
    int dtype_size;
    switch (args[ARG_dtype].u_int) {
```

**Review by iabdalkader — CHANGES_REQUESTED** [review]

**Comment by iabdalkader** [comment]
This function is broken, things work but model has no output. After wasting a lot of time trying to figure out what's wrong with it, I just copied the old input_callback from py_ml.c

**Comment by kwagyeman** [comment]
Sorry, I was trying to finish the testing code. I will fix that now.

**Comment by iabdalkader** [comment]
I think the current function is working correctly, and it's easier to read. There's no need to change it.

**Pull request closed without merging**